content/articles/2018-06-bench-1.0.1.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: bench 1.0.1
+date: '2018-06-28'
+slug: bench-1.0.1
+author: Jim Hester
+categories: [package]
+description: >
+  bench 1.0.1 is now available on CRAN. bench allows you to benchmark
+  code, tracking execution time, memory allocations and garbage
+  collections.
+photo:
+  url: https://unsplash.com/photos/nIkuMWT4Imc
+  author: Rula Sibai
+---
+
+[bench] is now available on CRAN!
+
+The goal of [bench] is to benchmark code, tracking execution time,
+memory allocations and garbage collections.
+
+Install the latest version with:
+``` r
+install.packages("bench")
+```
+
+### Comparison vs existing methods
```

Comment:
I think this should come after the usage example, and use "Compared to" instead of "Comparison vs"